### PR TITLE
update get external users api

### PR DESCRIPTION
**Pull request by emmaacha**

### CODE REVIEW

**Review by jatnerubia — CHANGES_REQUESTED** [review]
move the logic to external user service

**Comment by emmaacha** [comment]
> move the logic to external user service

Which logic kuys @jatnerubia ?

File: src/controllers/admin/external-user-controller.ts [hunk 1]
```diff
@@ -4,12 +4,27 @@ import { createExternalUserSchema } from "../../utils/validation/external-user-s
...
+    const { name, company, role, page } = req.query
+
+    const externalUsers = await ExternalUserService.getAllByFilters(
+      name as string,
+      company as string,
+      role as string,
+      page as string
+    )
+
+    res.json({
```
Comment (jatnerubia) [inline src/controllers/admin/external-user-controller.ts:24]: I think you can do `res.json(externalUsers)` here 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/controllers/admin/external-user-controller.ts (hunk 1 above)
Comment (emmaacha) [reply src/controllers/admin/external-user-controller.ts]: Oh my bad, updated it already tysm!

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**